### PR TITLE
chore: release 3.3.0

### LINKED ISSUE #183: Does not create packages installable on openSUSE
This issue has been fixed already in #182 - opening an issue to request a new release with this fix.

**What version of `electron-installer-redhat` are you using?**

3.2.0

**What version of `node` and `npm` are you using?**

node 15.8.0, npm 7.5.2

**What operating system are you using?**

Ubuntu 20.04 to create RPM, openSUSE Leap 15.2 to install

**Can you compile the [example app](https://github.com/unindented/electron-installer-redhat/tree/master/example) successfully?**

Yes

**What did you do? Please include the configuration you are using for `electron-installer-redhat`.**

Create any package and use `electron-installer-redhat` to create an RPM. Configuration n/a since this is a general issue

**What did you expect to happen?**

It is installable on openSUSE.

**What actually happened?**

It is not installable, due to the bug present before #182.

### PR DESCRIPTION
**Pull request by fcastilloec**
fixes #183

### CODE REVIEW

**Comment by fcastilloec** [comment]
@malept if this PR is approved. I will push the tag and release to npm

File: NEWS.md [hunk 1]
```diff
@@ -2,7 +2,15 @@
 
 ## [Unreleased]
 
-[Unreleased]: https://github.com/electron-userland/electron-installer-redhat/compare/v3.2.0...master
+[Unreleased]: https://github.com/electron-userland/electron-installer-redhat/compare/v3.3.0...master
+
+## [3.3.0] - 2021-02-02
```
Comment (malept) [inline NEWS.md:7]:
```suggestion
## [3.3.0] - 2021-02-05
```

**Review by malept — APPROVED** [review]

**Comment by fcastilloec** [comment]
@malept Just push the tag and also publish to npm

**Comment by davidbailey00** [comment]
Thanks!

**Pull request merged**